connect/api.go

```diff
@@ -44,3 +44,17 @@ func GetActivations(creds Credentials) (map[string]Activation, error) {
 	}
 	return activeMap, nil
 }
+
+func GetProduct(productQuery Product, creds Credentials) (Product, error) {
+	urlSuffix := "/connect/systems/products"
+	resp, err := DoGETWithParams(creds, urlSuffix, productQuery.toQuery())
```

Comment:
If I checkout this commit,  DoGETWithParams does not exist.

Reply:
Bad rebase. Fixed. Thx!